app/views/admin/project_steps/_duplicate_step_modal.html.slim

```diff
@@ -9,21 +9,27 @@ div.modal.fade.duplicate-step id="duplicate-step-modal-#{step.id}" tabindex="-1"
           = t("project_step.duplicate_step")
 
       div.modal-body
-        = form_tag do |f|
+        = form_tag("/admin/project_steps/#{step.id}/duplicate_step", method: 'post') do |f|
```

Comment:
Please use route helpers here. So `duplicate_admin_project_step_path(step)` (I think that should work).